web/themes/custom/votegov/src/sass/components/quick-links.scss

```diff
@@ -11,28 +11,59 @@
   @include grid-container;
   @include grid-row;
   @include vote-block-padding-y;
+  @include u-padding-x(3);
   @include u-flex('align-center');
+
+  @include at-media('tablet') {
+    @include u-padding-x(4);
+  }
+
+  @include at-media('desktop') {
+    @include u-padding-x(10);
+    @include u-padding-y(8);
+  }
+
 }
 
 .vote-quicklinks__heading {
   @include u-font-size('body', 'xl');
+  @include u-text('bold');
 
   h2 {
     color: $base-white;
   }
 
   @include at-media('desktop') {
-    @include grid-col(5);
+    @include grid-col(6);
   }
 }
 
 .vote-quicklinks__content{
   @include u-font('sans', 'md');
+  @include u-text('normal');
```

Comment:
You can remove this, the inherited style is the same.

Reply:
line removed